script.js

```diff
@@ -183,4 +183,22 @@ listProject.forEach((project) => {
   work.append(detailContainer);
 
   listWork.append(work);
+});
+
+/* Form Validation */
+const form = document.getElementById('form');
+const error = document.getElementById('error');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const { email } = form.elements;
+  if (email.validity.typeMismatch
+    || email.value.toLowerCase() !== email.value
+    || !email.validity.valid) {
+    error.textContent = 'Invalid email. Please ensure you have the correct email address; also, check that the content of the email field is in lowercase. Thank you.';
+    email.setCustomValidity('email is should be in lower case');
+    email.reportValidity();
+  } else {
+    email.setCustomValidity('');
+    form.submit();
+  }
```

Comment:
Good job 👌but here is a tricky bug with your form validation. When I entered a valid email for the first time it worked properly and the form is submitted. But when I entered an email with uppercase letters it throws an error (Which is expected), but here is when the problem occurs. After entering all of the lowercase letters again it still throws the same error, and the form is not being submitted. 

For simulation:
1. Enter an email with an uppercase letter and submit the form to see the error
2. Replace the uppercase letter with a lowercase letter and submit the form again (You'll see the form is not being submitted, and the error persists)

Please make sure to find and fix the bug.

![image](https://user-images.githubusercontent.com/34779928/202285109-2dfa70a8-ebb6-4699-ace5-2b1f169eecc5.png)